tests/mode.test.fixtures/00-tasks.tokens

```diff
@@ -0,0 +1,82 @@
+#0 <header>: *** Test Cases ***
```

Comment:
hm... can the line numbers be inferred? Seems like a lot.

Further... and not suggesting this, but dug up some of my [old work](
https://github.com/gtri/irobotframework/pull/7/files#diff-b25999ee0544e7d12540f110c7ef53a6e5a04fa44522597b0f03e1ab6155f9ef).

Looks like i was getting away with just having a stream of tokens, not caring about line breaks at all. Must have seemed like a good idea at the time! I do recall them being annoying to generate, as there is no context. I believe i just ran them through the test, let them fail, then inspected them by hand, as I was dealing with 100s of exemplars.

Reply:
There is a native jest way for snapshot testing. Changed the tests to use expect(tokens).toMatchSnapshot()
> hm... can the line numbers be inferred? Seems like a lot.
In this case, it's easier to find an original line when a test starts to fail.